packages/whats-new/src/style.scss

```diff
@@ -0,0 +1,197 @@
+// @TODO: remove the ignore rule and replace font sizes accordingly
+/* stylelint-disable scales/font-size */
+
+@import '~@automattic/typography/styles/fonts';
+@import '~@automattic/onboarding/styles/mixins';
+
+$wpcom-modal-breakpoint: 660px;
+
+$wpcom-modal-padding-v: 30px;
+$wpcom-modal-padding-h: 25px;
+$wpcom-modal-content-min-height: 350px;
+$wpcom-modal-footer-padding-v: 20px;
+$wpcom-modal-footer-height: 30px + ( $wpcom-modal-footer-padding-v * 2 );
+
+// Core modal style overrides
+.whats-new {
+	&.components-modal__frame {
+		overflow: visible;
+		height: 65vh;
+		top: calc( 17.5vh - #{$wpcom-modal-footer-height / 2} );
+
+		@media ( max-width: $wpcom-modal-breakpoint ) {
+			width: 90vw;
+			min-width: 90vw;
+			left: 5vw;
+			right: 5vw;
+		}
+
+		@media ( min-width: $wpcom-modal-breakpoint ) {
+			width: 720px;
+			height: $wpcom-modal-content-min-height;
+			top: calc( 50% - #{$wpcom-modal-footer-height / 2} );
+		}
+	}
+
+	.components-modal__header {
+		display: none;
+	}
+
+	.components-guide__container {
+		margin-top: 0;
+	}
+
+	.components-guide__footer {
+		position: absolute;
+		width: 100%;
+		height: $wpcom-modal-footer-height;
+		bottom: $wpcom-modal-footer-height * -1;
+		left: 0;
+		padding: $wpcom-modal-footer-padding-v 0;
+		margin: 0;
+		display: flex;
+		justify-content: center;
+		background: white;
+		border-top: 1px solid #dcdcde;
+
+		@media ( min-width: $wpcom-modal-breakpoint ) {
+			border-top: none;
+		}
+	}
+
+	.components-guide__page {
+		position: absolute;
+		width: 100%;
+		height: 100%;
+		justify-content: start;
+	}
+
+	.components-guide__page-control {
+		position: relative;
+		height: 0;
+		top: 100%;
+		overflow: visible;
+		margin: 0 auto;
+		z-index: 2;
+
+		&::before {
+			display: inline-block;
+			content: '';
+			height: $wpcom-modal-footer-height;
+			vertical-align: middle;
+		}
+
+		li {
+			vertical-align: middle;
+			margin-bottom: 0;
+		}
+
+		// Temporarily disable dots on mobile as alignment is wonky.
+		display: none;
+		@media ( min-width: $wpcom-modal-breakpoint ) {
+			display: block;
+		}
+	}
+}
+
+.whats-new-page__container {
+	display: flex;
+	flex-direction: column-reverse;
+	justify-content: flex-end;
+	background: white;
+	width: 100%;
+	height: 100%;
+
+	@media ( min-width: $wpcom-modal-breakpoint ) {
+		flex-direction: row;
+		justify-content: flex-start;
+		position: absolute;
+		min-height: $wpcom-modal-content-min-height;
+		bottom: 0;
+	}
+}
+
+.whats-new-page__text,
+.whats-new-page__visual {
+	@media ( min-width: $wpcom-modal-breakpoint ) {
+		flex: 1 auto;
+		min-width: 290px;
+	}
+}
+
+.whats-new-page__text {
+	padding: 0 25px 25px;
+	height: 60%;
+
+	@media ( min-width: $wpcom-modal-breakpoint ) {
+		height: auto;
+		padding: $wpcom-modal-padding-v $wpcom-modal-padding-h;
+	}
+}
+.whats-new-page__visual {
+	height: 40%;
+	padding: 25px 25px 0;
+	background: #1581d8;
```

Comment:
Using colors like this isn't recommended and won't guarantee that we consider the user's current color scheme. Could we use variables for all the colors instead?